test/services/assessmentcentre/AssessmentCentreParsingServiceSpec.scala

```diff
@@ -0,0 +1,44 @@
+package services.assessmentcentre
+
+import model.persisted.assessmentcentre.Event
+import services.BaseServiceSpec
+
+import scala.concurrent.Future
+
+/**
+  * Created by fayimora on 08/06/2017.
+  */
+class AssessmentCentreParsingServiceSpec extends BaseServiceSpec {
+  "processCentres" must {
+    "successfully saves and loads the file contents" in new GoodTestFixture {
+      val events: Seq[Event] = service.processCentres().futureValue
+
+      events.size mustBe 2
+      events.head.eventType mustBe "FSAC"
+      events(1).skillRequirements.getOrElse("QUALITY_ASSURANCE_COORDINATOR", "--") mustBe 0
+    }
+
+    "fails gracefully when any field is malformed" in new MalformedTestFixture {
+      an[Exception] must be thrownBy service.processCentres().futureValue
```

Comment:
@benjstephenson At the moment, I'm simply returning `Failure[Exception]`. I got rid of the multiple cases for each type of exception